web/pimcore/static6/js/pimcore/asset/listfolder.js

```diff
@@ -148,7 +148,13 @@ pimcore.asset.listfolder = Class.create({
             tbar: [
                 "->"
                 ,this.checkboxOnlyDirectChildren
-                ]
+                , "-"
+                ,{
+                    text: t("download_as_zip"),
+                    iconCls: "pimcore_icon_export",
```

Comment:
The class `pimcore_icon_zip pimcore_icon_overlay_download` would be more consistent I'd say.